docs/source/API/core/policies/MDRangePolicy.rst

```diff
@@ -86,7 +86,55 @@ Constructors
 
     * Provide a start and end index as well as the tiling dimensions. The length of the lists must match the rank of the policy.
 
+CTAD Constructors (since 4.3):
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: cppkokkos
+
+   DefaultExecutionSpace des;
+   SomeExecutionSpace ses; // different from DefaultExecutionSpace
+   int64_t i;
+
+   // Deduces to MDRangePolicy<Rank<5>>
+   MDRangePolicy pl0({1, 2, 3, 4, 5}, {1, 2, 3, 4, 5});
+   MDRangePolicy pl1({1, 2, 3, 4, 5}, {1, 2, 3, 4, 5}, { i });
```

Comment:
Is this legal? If tiles are provided, they mist be specified for all ranks.

Reply:
I don't think that's true.

If you look at the non-CTAD test at https://github.com/kokkos/kokkos/blob/1d9d0df2eecfce635fe5c77559eb17adfa128d04/core/unit_test/TestMDRangePolicyConstructors.hpp#L54, the tiling is a `Kokkos::Array` of dimension 1.

While CTAD has its own overload set, they all have to map to existing constructors.

Reply:
Hmm... does that make sense, though?